lib/src/widgets/auth_card.dart

```diff
@@ -790,6 +793,7 @@ class _RecoverCardState extends State<_RecoverCard>
       prefixIcon: Icon(FontAwesomeIcons.solidUserCircle),
       keyboardType: TextInputType.emailAddress,
       autocorrect: false,
+      autofillHints: [AutofillHints.email],
```

Comment:
Same here `AutofillHints.username`

Reply:
It should also have AutofillHints.newUsername for the registration view.

If you look at the registration form, could you also address #54? 

I suck at Flutter, otherwise i would help you accomplish that...

Reply:
Check my other answer, check if that makes sense.

Reply:
Same: https://github.com/NearHuscarl/flutter_login/pull/125#discussion_r605101707